jena-arq/src/main/java/org/apache/jena/riot/lang/rdfxml/SysRRX.java

```diff
@@ -29,8 +29,21 @@
  */
 public class SysRRX {
 
+    /**
+     * Creates and initializes a javax.xml.stream.XMLInputFactory#newDefaultFactory().
+     * @return
```

Comment:
```suggestion
     * @return XMLInputFactory
```

Reply:
I recognized, that I kept using `javax.xml.stream.XMLInputFactory#newInstance()` instead of `javax.xml.stream.XMLInputFactory#newDefaultFactory()`, so I fixed the javadoc accordingly.